.circleci/config.yml

```diff
@@ -191,6 +220,13 @@ workflows:
               only: /^v.*/
             branches:
               ignore: /.*/
+      - release-chart:
+          context: telepresence2-release
+          filters:
+            tags:
+              only: /^chart-v.*/
```

Comment:
1. Do we expect to be doing chart releases that don't coincide with a Telepresence release?  IMO, this would be a good opportunity to version them in lockstep.
2. It would be good to just modify the regex here instead of having the `[[ $CIRCLE_TAG == … ]]` check in the job.  I believe `/^chart-v(.(?!rc))*$/` would do what you want (the Circle [docs](https://circleci.com/docs/2.0/workflows/#using-regular-expressions-to-filter-tags-and-branches) say that it uses [`java.util.regex` syntax](https://docs.oracle.com/javase/7/docs/api/java/util/regex/Pattern.html)).
3. So using my suggestion of excluding all prerelease tags, the regex would be `/^chart-v[^-]*$/`

Reply:
1. So in an ideal world, no. and I had originally planned to keep the versions in-sync. But after chatting with people on edge stack, they noted the possibility that sometimes you might need to only change the chart (e.g. an error in the yaml when a user uses some value, which isn't used in our smoke tests).  

Which maybe for edge stack that is a problem since they release less frequently, but we could do another release shortly after the problem was discovered "x.y.z+1" with a changelog that's like, "Bugfix: Fixes issue with helm chart".  So if we aren't opposed to doing something like that, then I think for our own sanity it makes sense to keep the versions in lockstep.

Reply:
1. Given the goal of keeping telepresence releases light, I have no problem "Bugfix: Helm chart" releases.  So in my mind, that means not having separate `chart-v*` tags.

Reply:
Sounds good, I will remove the `chart-v*` and fold helm chart publish into our already existing release steps then.